### DISCUSSION TITLE
Turn auto-login on for LXC created with root password?

### DISCUSSION THREAD

**Discussion by ThiSaadeh** [tteck/Proxmox#2558] [Q&A] [answered]
I thought it was originally a good idea to setup a ssh root user/password instead of using the auto-login from the scripts, but then after using tteck scripts for a while I ended up realizing I'd much prefer to have the automatic-login for console than having to enter user/password every time I need console access via Proxmox Console GUI. 

Is it possible via commands to go back to using auto-login and disable the root authentication?

Thanks in advance for the crazy good scripts and help.

**Comment by tteck** [comment, marked as answer]
yes, copy/paste, hit enter
```
GETTY_OVERRIDE="/etc/systemd/system/container-getty@1.service.d/override.conf"
mkdir -p $(dirname $GETTY_OVERRIDE)
cat <<EOF >$GETTY_OVERRIDE
  [Service]
  ExecStart=
  ExecStart=-/sbin/agetty --autologin root --noclear --keep-baud tty%I 115200,38400,9600 \$TERM
EOF
systemctl daemon-reload
systemctl restart $(basename $(dirname $GETTY_OVERRIDE) | sed 's/\.d//')
```

**Reply by ThiSaadeh** [reply]
You're a genius, that's really handy, I'm saving it for the future.

**Reply by jtcleek** [reply]
Can you provide the steps for an Alpine LXC?

**Reply by enter360** [reply]
Agreed. Can you please provide the steps for Alpine LXC ?

**Reply by netgabo** [reply]
Genius

**Reply by tteck** [reply]
Alpine:

```
bash -c "passwd -d root"
```
Still have to login: `root` but no password is needed.
